src/garage/_dtypes.py

```diff
@@ -167,3 +166,91 @@ def __new__(cls, env_spec, observations, actions, rewards, terminals,
         return super().__new__(TrajectoryBatch, env_spec, observations,
                                actions, rewards, terminals, env_infos,
                                agent_infos, lengths)
+
+
+class TimeStep(
```

Comment:
when manually indenting code and comments, please use the classic terminal style of justified wrapping -- that is, making maximum use of horizontal space before wrapping a line.

Reply:
resolved